newsletter-june-2023.rst

```diff
@@ -0,0 +1,71 @@
+.. post:: June 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+   :category: Newsletter
+
+Read the Docs newsletter - June 2023
+====================================
+
+News and updates
+----------------
+
+- ⚠️ A ``.readthedocs.yaml`` configuration file will be required for your future builds.
+  Read more about this change in :doc:`/migrate-configuration-v2`.
+- ✅️ Visiting ``/en/`` on a documentation project now redirects to ``/en/latest/``.
```

Comment:
This is true for any language, right? Should note that more explicitly.

Reply:
True :+1: Fixed that one, it took a few more words to explain though :)